dep.pl

```diff
@@ -45,7 +45,7 @@
       print {$out} $line;
    } else {
       print {$out} << "EOS";
-#include <tommath.h>
+#include "tommath_private.h"
```

Comment:
shouldn't this be `#include "tommath.h"`?

Reply:
oh, the patch you added is apparently correct :)

thx!

Reply:
That is the include-line for new files and the new files should get the same treatment as the old files and include `tommath_private.h` instead of `tommath.h`. 

My perl knowledge is a bit rusty and it took me some time to parse through that file but it should be the correct place.

Reply:
(Why do I have to refresh to see your second comment but the mail came?)
Second try of build 149.7 was successful, merge?

Reply:
> (Why do I have to refresh to see your second comment but the mail came?)

good question, probably @github can help? :D